src/renderer/utils/LiveryConversion.ts

```diff
@@ -281,6 +277,9 @@ export class LiveryConversion {
         const parser = new ConfigIniParser();
 
         const modelCfgPath = path.join(packageFolder, 'SimObjects', 'AirPlanes', simObjectName, `MODEL.${modelFolderName}`, 'model.cfg');
+        if (!existsSync(modelCfgPath)) {
+            return null;
```

Comment:
No need to return anything, it's a void function

Reply:
@FoxtrotSierra6829

Reply:
resolved @Benjozork